community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/input/csv/InputEntityDeserializer.java

```diff
@@ -176,6 +186,11 @@ private boolean deserializeNextFromSource() throws IOException
     @Override
     public void close()
     {
+        if ( ignoredColumns > 0 )
+        {
+            System.out.println( String.format( "Warning: Ignored %d extra columns in %s", ignoredColumns,
```

Comment:
Having tested both, I vote for @tinwelint 's suggestion. Seems to be the most useful. Example from the test with variable number of extra columns:

```
>:??--------------------------|PROPERTI|NODE:7.63 MB|*LABEL SCAN----------------------------|v] 10k
Done in 292ms
Warning: Ignored several extra columns in 
    /home/jonas/workspace/2.2/community/import-tool/target/test-data/org.neo4j.tooling.ImportToolTest/graph-db/1-nodes.csv
If this is unexpected, check the headers. Example of ignored values:
Row 1: 	ExtraColumn0	ExtraColumn1
Row 2: 	ExtraColumn0	ExtraColumn1
Row 3: 	ExtraColumn0
Row 4: 	ExtraColumn0
Row 7: 	ExtraColumn0
Row 8: 	ExtraColumn0	ExtraColumn1
Row 10: 	ExtraColumn0
Row 11: 	ExtraColumn0
Row 13: 	ExtraColumn0

Prepare node index

[*DETECT:7.63 MB-------------------------------------------------------------------------------]   0
```